src/scss/_variables.scss

```diff
@@ -30,11 +30,12 @@ $_o-forms-field-default-font-size: 14px;
 /// Spacing variables.
 /// @access private
 /// @type String
-$_o-forms-group-spacing: oTypographySpacingSize(5); // 20px
-$_o-forms-inline-group-spacing: oTypographySpacingSize(8); // 32px
```

Comment:
I think renaming variables makes this a breaking change

Reply:
They're marked private (sassdoc and underscore) so a rename is okay (in theory).